src/components/NewsRoom.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { IconHeading } from '@baltimorecounty/dotgov-components';
+import NewsGroup from './NewsGroup';
+const News = [
+	{
+		id: 1,
+		date: '07/29/2019',
+		headline: 'Baltimore County Marks Progress in Rehabilitation of Winters lane Homes',
+		snippet: 'Today Baltimore County officials joined with community and...',
+		link: '//www.baltimorecountymd.gov/News/BaltimoreCountyNow'
+	},
+	{
+		id: 2,
+		date: '07/29/2019',
+		headline: 'Baltimore County Marks Progress in Rehabilitation of Winters lane Homes',
+		snippet: 'Today Baltimore County officials joined with community and...',
+		link: '//www.baltimorecountymd.gov/News/BaltimoreCountyNow'
+	},
+	{
+		id: 3,
+		date: '07/29/2019',
+		headline: 'Baltimore County Marks Progress in Rehabilitation of Winters lane Homes',
+		snippet: 'Today Baltimore County officials joined with community and...',
+		link: '//www.baltimorecountymd.gov/News/BaltimoreCountyNow'
+	},
+	{
+		id: 4,
+		date: '07/29/2019',
+		headline: 'Baltimore County Marks Progress in Rehabilitation of Winters lane Homes',
+		snippet: 'Today Baltimore County officials joined with community and...',
+		link: '//www.baltimorecountymd.gov/News/BaltimoreCountyNow'
+	}
+];
+
+const NewsRoom = () => (
+	<React.Fragment>
+		<IconHeading text="Latest Headlines" icon="fa fa-newspaper" />
```

Comment:
The grid is duplicated in the [NewsRoomSection](https://github.com/baltimorecounty/dotgov-prototype-2.0/pull/12/files#diff-b406ca93c1250fab5c0a16262164cd42) component.

What I would recommend is moving all of the grid stuff to your section component, except in src/components/NewsGroup.js 

any section is going to look like this

```jsx
<Section>
  <div className="container">
    <div className="row">
      <div className="col">
         <IconHeading />
      </div>
    </div>
    <div className="row">
      { /** Your comopnent cols go here */}
    </div>
  </div>
</Section>
```

If you want, you could create a helper component that does this automatically or we can add that in the future.

Reply:
done